google/resource_storage_bucket_object.go

```diff
@@ -79,6 +79,26 @@ func resourceStorageBucketObject() *schema.Resource {
 			},
 
 			"md5hash": &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+
+			"predefined_acl": &schema.Schema{
+				Type:     schema.TypeString,
+				Removed:  "Please use resource \"storage_object_acl.predefined_acl\" instead.",
+				Optional: true,
+				ForceNew: true,
+			},
+
+			"source": &schema.Schema{
+				Type:          schema.TypeString,
+				Optional:      true,
+				ForceNew:      true,
+				ConflictsWith: []string{"content"},
+			},
+
+			// Detect changes to local file or changes made outside of Terraform to the file stored on the server.
+			"detect_md5hash": &schema.Schema{
 				Type:     schema.TypeString,
 				Optional: true,
```

Comment:
Does this need to be Optional (vs computed-only)? If so can you add a comment why?

Reply:
Computed does not trigger a diff

Reply:
It does if the field has a value